torchgeo/datasets/utils.py

```diff
@@ -485,7 +486,10 @@ def rgb_to_mask(
 
 
 def percentile_normalization(
-    img: np.ndarray, lower: float = 2, upper: float = 98  # type: ignore[type-arg]
+    img: np.ndarray,  # type: ignore[type-arg]
+    lower: float = 2,
+    upper: float = 98,
+    axis: Optional[Union[int, Tuple[int, int]]] = None,
```

Comment:
```suggestion
    axis: Optional[Union[int, Sequence[int]]] = None,
```
I think both lists and tuples of any length are valid.